### DISCUSSION TITLE
2023년 10월 03일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#85] [General]
# 2023년 10월 03일

TODO 리뷰 반영하기  [priority:: high]  [due:: 2023-10-02]  [completion:: 2023-10-03] 링글 복습하기  [priority:: medium]  [due:: 2023-10-02] 닉네임 변경 PR 보내기  [priority:: highest]  [due:: 2023-10-03] LOG 배운 것 느낀 것

https://vero.wiki/2023-10-03/

**Comment by kevstevie** [comment]
베로위키 리뉴얼했나요

**Reply by Cyma-s** [reply]
이게 더 깔끔하죠
